### DISCUSSION TITLE
Game save location?

### DISCUSSION THREAD

**Discussion by neffo** [Heroic-Games-Launcher/HeroicGamesLauncher#513] [Q&A] [answered]
I was hoping to migrate my game saves from Windows for Subnautica, but can't locate where Heroic and or Proton creates the save files, and clues?

**Comment by flavioislima** [comment, marked as answer]
They stay on your prefix. Imagine your prefix, as being your Windows instalation. So search for the same folder you have in Windows but this time starting from the prefix.
For instance.
If on windows the save folder is on c:\\users\user\appdata\game
If your prefix is:
.wine
The save folder will be:
.wine\drive_c\users\user\appdata\game.

A few things to consider:
If your using proton your prefix will be in a folder called pfx inside the prefix you choose.
And your username will be always steamuser.
So If your prefix is .wine, with proton will be:
.wine/pfx.

Then just follow the same logic to find the save folder.

You can find the game prefix under the game settings in Heroic.

**Reply by neffo** [reply]
Perfect thanks Flavio!
